### PR TITLE
GitHub Action

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#24 
#### What does this implement/fix? Explain your changes.
- Test system modified

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      run: |
+        python -m pytest test --cov=nafas --cov-report=term
+    - name: Other tests
+      run: |
+        python -m vulture nafas/ setup.py --min-confidence 65 --exclude=__init__.py --sort-by-size
+        python -m bandit -r nafas -s B311
+        python -m pydocstyle --match-dir=nafas -v
+    - name: Codecov
+      run: |
+        codecov
```
Comment (sadrasabouri) [inline .github/workflows/test.yml:46]: new line may be forgotten here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `time_convert` function
 ### Changed
 - Menu optimized
+- Test system modified
```
Comment (sadrasabouri) [inline CHANGELOG.md:23]: Is this change important for user?

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Thank you :100: 
I left some comments for you, hope they'll be useful.

File: .github/workflows/test.yml (hunk 1 above)
Comment (sepandhaghighi) [reply .github/workflows/test.yml]: solved in https://github.com/sepandhaghighi/nafas/pull/25/commits/5e4c0f44d37b4cdbafdc9200ec1f66ccc591db44

File: CHANGELOG.md (hunk 2 above)
Comment (sepandhaghighi) [reply CHANGELOG.md:23]: I think it would be good if we mention it like this.

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**